css-three-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container {
+  width: 80%;
+  margin: auto;
+  background-color: #1996da;
+  padding-right: 10rem;
+  padding-top: 3rem;
+  padding-left: 10rem;
+  padding-bottom: 2rem;
+}
+
+.column {
+  display: block;
```

Comment:
An unusual choice of styling here, your `column` class elements are `div` elements, which means they're already `block`. A `column` class is meant to have only styling associated with setting a size for an element, you likely should have something like `width: 33%` here